src/model/TestCase.ts

```diff
@@ -19,6 +19,10 @@ export class TestCase {
         return this.dimensions.get( name ) || '';
     }
 
+    isDesktop(): boolean {
+        return this.dimensions.has('resolution');
```

Comment:
```suggestion
        return !this.dimensions.has('device');
```

While resolution works for now, I'd prefer if we identified desktop/mobile/tablet channels better. The best way of doing this would be to add the identifiers to the metadata in our `banner-screenshots` tool. The second-best would be to look for the `device` dimension (only mobile simulators have a "device").